README.md

```diff
@@ -1 +1,75 @@
-# Stock-Market
\ No newline at end of file
+# Stock-Market
+
+## Deployment
+
+(Link will be provided after finishing the project)
```

Comment:
- [ ] As required for the project, Kindy make sure you add a **new section** in your README file of a **Loom** video where you present your project. And the video should have the following features:
  - The video should present the features of the project.
  - The maximum length of the video is 5 minutes.
  - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
  - The video contains a demo of the project, highlighting all the project features.
  - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.